README.md

```diff
@@ -236,12 +237,18 @@ For more, see [Awesome-cl](https://github.com/CodyReichert/awesome-cl).
 
 
 ### Internet
+
 #### Browser
 
 - [nEXT](https://github.com/nEXT-Browser/nEXT) -  nEXT - The fastest productivity web-browser.
 
 See also [Closure](https://github.com/dym/closure) (using McClim) and [Lispkit](https://github.com/AeroNotix/lispkit) (using WebKit).
 
+#### Torrent
+
+- [cl-torrents](https://github.com/vindarel/cl-torrents)
+  - a library and app with a command line and readline interface to search for torrents on popular trackers.
```

Comment:
I think you write the wrong format :).

See [https://github.com/azzamsa/awesome-cl-software#guidelines-to-contribute](https://github.com/azzamsa/awesome-cl-software#guidelines-to-contribute)